tsl/src/nodes/decompress_chunk/planner.c

```diff
@@ -377,27 +377,67 @@ find_attr_pos_in_tlist(List *targetlist, AttrNumber pos)
 }
 
 static bool
-qual_is_vectorizable(DecompressChunkPath *path, Node *qual)
+contains_volatile_functions_checker(Oid func_id, void *context)
+{
+	return (func_volatile(func_id) == PROVOLATILE_VOLATILE);
+}
+
+static bool
```

Comment:
This function contains some logic. So, could you add a comment?

Reply:
+1 I believe you are just looking for expressions that can be constified at runtime (as opposed to plantime) here. It would help to add a comment.

I was thrown off by the commit message that said we can now support `ts> now() -  '1 day'::interval.` We already do plan time constification for that in some cases. I suggest also modifying the commit message to elaborate that this change now introduces support for  ts < now()  , ts> now() etc.

Reply:
Thanks. To clarify: ts > now() - <> works with plantime chunk exclusion. But I wasn't aware that it doesn't get passed along via the filters to decompress.